src/main/java/Task.java

```diff
@@ -0,0 +1,37 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description, boolean isDone) {
```

Comment:
I think that the constructor for a Task doesn't need to have the isDone argument since when a Task is first created it should be initialised as not done